chapter05/Blackjack.py

```diff
@@ -201,20 +201,21 @@ def monteCarloOnPolicy(nEpisodes):
 def monteCarloES(nEpisodes):
     # (playerSum, dealerCard, usableAce, action)
     stateActionValues = np.zeros((10, 10, 2, 2))
-    stateActionPairCount = np.zeros((10, 10, 2, 2))
+    stateActionPairCount = np.ones((10, 10, 2, 2))
```

Comment:
It's ok to use ones because it doesn't influence relative relationship between different actions. But it would be better to write a comment here like I did before.